internal/v4/api.go

```diff
@@ -614,8 +614,10 @@ func (h *ReqHandler) metaTags(entity *mongodoc.Entity, id *router.ResolvedURL, p
 // GET id/meta/stats/
 // https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetastats
 func (h *ReqHandler) metaStats(entity *mongodoc.Entity, id *router.ResolvedURL, path string, flags url.Values, req *http.Request) (interface{}, error) {
+
 	// Retrieve the aggregated downloads count for the specific revision.
-	counts, countsAllRevisions, err := h.Store.ArchiveDownloadCounts(id.PreferredURL())
+	_, forceRefreshPresent := flags["force_refresh"]
```

Comment:
I think we should treat this as a boolean parameter like any other.
Also, how about just "refresh" rather than "force_refresh" ?
For example:

    			refresh, err = router.ParseBool(flags.Get("refresh))
			if err != nil {
				return charmstore.SearchParams{}, badRequestf(err, "invalid refresh parameter")
			}